apps/components-e2e/src/components/filter-field/filter-field.ts

```diff
@@ -51,6 +51,19 @@ const TEST_DATA = {
         },
       ],
     },
+    {
+      name: 'Autocomplete with free text options',
+      autocomplete: [
+        'Autocomplete option 1',
```

Comment:
please rebase with current changes on master string as node is not supported anymore. Please use an object with name: `{ name: 'Autocomplete option 1' }`